src/server/editorServices.ts

```diff
@@ -901,22 +901,22 @@ module ts.server {
             }
             return false;
         }
-        
+
         openConfigFile(configFilename: string, clientFileName?: string): ProjectOpenResult {
             configFilename = ts.normalizePath(configFilename);
             // file references will be relative to dirPath (or absolute)
             var dirPath = ts.getDirectoryPath(configFilename);
-            var rawConfig = <ProjectOptions>ts.readConfigFile(configFilename);
-            if (!rawConfig) {
-                return { errorMsg: "tsconfig syntax error" };
+            var rawConfig: { config?: ProjectOptions; error?: Diagnostic; } = ts.readConfigFile(configFilename);

```

Comment:
The root issue of Microsoft/TypeScript-Sublime-Plugin#180 seems to be I accidentally uploaded a locally built tsserver.js, and the problem went away with the official 1.5 beta one, so it may not apply here.